sandpack-react/src/components/FileTabs/index.tsx

```diff
@@ -29,6 +46,11 @@ export const FileTabs: React.FC = () => {
             type="button"
           >
             {getFileName(filePath)}
+            {closableTabs && openPaths.length > 0 ? (
```

Comment:
In order to improve the user experience, I wouldn't render the close button when it's a single child. Based on your screen recording and internal test, actually you can close it where there is only one tab item. Does it make sense?

Reply:
I mean just to keep it uniform i let the close-icon. But actually even if clicked, it will not close the tab. I will remove the icon too 👍